docs/standard/native-interop/best-practices.md

```diff
@@ -190,6 +190,12 @@ handle.Free();
 
 Don't forget that `GCHandle` needs to be explicitly freed to avoid memory leaks.
 
+### Advanced managed wrapper management
```

Comment:
Nit: I do not consider SafeHandles to be an advanced solution. I consider it the basic go-to solution. Finalizers are the advanced topic.